README.md

```diff
@@ -51,15 +51,14 @@ We'd love your help!. Use tags [up-for-grabs][up-for-grabs-issues] and
 
 ## Useful links
 
-- For more information on OpenTelemetry, visit: <https://opentelemetry.io/>
+- For more information on OpenTelemetry, see [opentelemetry.io](https://opentelemetry.io).
 - For conventions used in OpenTelemetry traces, see the following [readme](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/README.md).
 - For help or feedback on this project, join us in [GitHub Discussions](https://github.com/open-telemetry/opentelemetry-erlang-contrib/discussions), `#otel-erlang-elixir` channel in the [CNCF slack](https://slack.cncf.io/), and `#opentelemetry` channel in the [Elixir Lang slack](https://elixir-slackin.herokuapp.com/)
 - Erlang SIG [community page](https://github.com/open-telemetry/community#special-interest-groups)
 
 ## License
 
-Apache 2.0 - See [LICENSE][license-url] for more information.
+Apache 2.0 - See [LICENSE](LICENSE) for more information.
 
 [discussions-url]: https://github.com/open-telemetry/opentelemetry-erlang-contrib/discussions
-[license-url]: https://github.com/open-telemetry/opentelemetry-erlang-contrib/blob/main/LICENSE
```

Comment:
I think this was used so the link works even from hexdocs.

Reply:
Oh, no, nm, this is the top level readme that isn't used when publishing any individual package.